socket/client/src/backends/native/socket.rs

```diff
@@ -42,12 +40,12 @@ impl Socket {
         );
         let conditioner_config = self.config.link_condition.clone();
 
-        let (addr_cell, to_server_sender, to_client_receiver) =
-            get_runtime().block_on(RTCSocket::connect(&server_session_string));
+        let (socket, io) = RTCSocket::new();
+        get_runtime().spawn(async move { socket.connect(&server_session_string).await });
```

Comment:
Here's a really important bit: we totally don't want to block a thread by waiting for the WebRTC session to initialise (which implies heavy IO).

Which is why https://github.com/naia-lib/webrtc-unreliable-client/pull/5 is required.